sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/agent-agent-role.json

```diff
@@ -1,14 +1,14 @@
 {
-  "identifier": "http://rs.tdwg.org/dwc/dwc-dp/table-schemas/agent-agent-role",
-  "url": "https://rs.gbif.org/sandbox/experimental/data-packages/dwc-dp/0.1/table-schemas/agent-agent-role.json",
+  "identifier": "http://rs.tdwg.org/dwc/dwc-dp/agent-agent-role",
+  "url": "table-schemas/agent-agent-role.json",
```

Comment:
If we change URLs like this IPT won't be able to upload table schemas

Reply:
@mike-podolskiy90 can you clarify what is needed?

1. Is the `identifier` (without `table-schemas`) ok?
2. Do you need the full URL in the `url` property?

For the latter: we dropped the full `url` so that we don't have to maintain a version number in each file in the [dwc-dp repo](https://github.com/gbif/dwc-dp/blob/master/dwc-dp/table-schemas/agent-agent-role.json). If the IPT requires them, they could be expanded to full URLs either:

- When dropping the files on rs.gbif.org/rs.tdwg.org. I assume currently manually done by @tucotuco in PRs like these, but in the future as part of a [build process](https://github.com/gbif/dwc-dp/issues/139).
- By the IPT

Reply:
I would encourage we always us absolute URLs to the versioned schemas (i.e. `https://rs.tdwg.org/dwc-dp/0.1/agent-agent-role.json`) for robustness and ease of use.

I can see the advantage of curating them as single files in git (i.e. git commit messages), although would also note that a `diff` across files is also pretty easy, and the build and release policies are more intuitive (i.e. a git pull is all that is required or a deployment, rather than a script known only to a few individuals). With our limited resources, I'd probably favour simplicity myself (i.e. folder in git per version).

Reply:
I meant the `url` field, identifiers are okay. IPT uses that field to download an actual table schema and all its fields

Reply:
I have made a local commit that reinstates full URLs again (point 3), but I don't have write access to push. @mike-podolskiy90 could you grant me write access to this repo?